kubeprod/cmd/install.go

```diff
@@ -52,6 +53,7 @@ func init() {
 
 	InstallCmd.PersistentFlags().String(FlagManifests, DefaultManifestBase(), "Base URL below which to find platform manifests")
 	InstallCmd.PersistentFlags().String(FlagPlatformConfig, prodruntime.DefaultPlatformConfig, "Path for generated platform config file")
+	InstallCmd.PersistentFlags().BoolP(FlagOnlyGenerate, FlagOnlyGenerateShort, false, "Whether to just generate the root manifest and configuration files")
```

Comment:
Somehow we need to communicate that this also does the actual non-k8s setup (ie: it is not a read-only operation).

Maybe: `"Stop before pushing configuration to the cluster"` ?